src/domain/validation/FlowContainerValidator.ts

```diff
@@ -14,6 +15,7 @@ export function getFlowStructureErrors(container: IContainer): ErrorObject<strin
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const flowSpecJsonSchema = require('../../../dist/resources/flowSpecJsonSchema.json')
   const ajv = new Ajv()
+  ajvFormat(ajv) // we need this to use AJV format such as 'date-time' (https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.7)
```

Comment:
FYI: @bzabos 
This was needed to correctly use json schema format. Solution from here: https://github.com/ajv-validator/ajv/issues/1295#issuecomment-754679968